lib/fluent/diagtool/diagutils.rb

```diff
@@ -14,15 +14,28 @@
 #    limitations under the License.
 #
 
+module Diagtool
+  ON_WINDOWS = /mingw/.match?(RUBY_PLATFORM)
```

Comment:
How about using `windows?` predicate?

```
module Diagtool
   def self.windows?
      /mingw/.match?(RUBY_PLATFORM)
   end
end
```

You can use `Diagtool.windows?` it is similar to `Fluent.windows?`.

Reply:
Is the advantage of that that it can be used the same way as Fluentd code?
I am concerned that this this value can be determined at initialization.

Reply:
How about this?

```ruby
module Diagtool
  ON_WINDOWS = /mingw/.match?(RUBY_PLATFORM)
  def self.windows?
    ON_WINDOWS
  end
end
```

Reply:
> I think this value can be determined at initialization.

I've overlooked this point of view.

so, suggested https://github.com/fluent/diagtool/pull/16#discussion_r1401469932 looks good.

Reply:
OK! Thanks for your feedback! I will change this code.

Reply:
Fixed!